README.md

```diff
@@ -0,0 +1,9 @@
+Important Information About This Repository
+
+HAL Espressif (or `hal_espressif`) is a collection of tools, low level APIs and hardware access functions for Espressif Devices.
+
+This repository is a copy (not a fork) of [esp-hal3rd party repository](https://github.com/espressif/esp-hal-3rdparty). If a SoC is not yet supported, work starts in `esp-hal-3rdparty`.
+
+For any support and feature request, please, refer to Zephyr's [main repository](https://github.com/zephyrproject-rtos/zephyr) instead.
```

Comment:
I'd suggest clarifying what "referring to Zephyr's repository" means.
Do you mean PR's or Issues should be open there? Possibly better to state it.